lib/nfg/capistrano/tasks/deploy.rake

```diff
@@ -25,11 +25,7 @@ namespace :deploy do
     if s3_assets_file.exists?
       FileUtils.mkdir_p "#{shared_path}/public/assets"
       on release_roles :all do
```

Comment:
This change is just replacing `aws s3api` with the simpler and more succinct `aws s3 cp`.